tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -254,6 +254,7 @@ public function testDescribeTable()
         $expected = [
             'id' => [
                 'type' => 'biginteger',
+                'baseType' => null,
```

Comment:
There is no actual base type in those cases, but I can definitely set it to the same as the current type